This is a quick hack for the following problem:

What alt-o used to do is now done by alt-i. This may be problematic for
upgraders, so just swap the two options. A better way to handle the
problem would be to introduce an option for that.

diff --git a/src/screen.c b/src/screen.c
index 6cac7b0..1e93c85 100644
--- a/src/screen.c
+++ b/src/screen.c
@@ -2639,7 +2639,7 @@ panel_execute_cmd (WPanel *panel, unsigned long command)
 
     switch (command) {
     case CK_PanelChdirOtherPanel:
-        chdir_other_panel (panel);
+        sync_other_panel (panel);
         break;
     case CK_PanelChdirToReadlink:
         chdir_to_readlink (panel);
@@ -2729,7 +2729,7 @@ panel_execute_cmd (WPanel *panel, unsigned long command)
         start_search (panel);
         break;
     case CK_PanelSyncOtherPanel:
-        sync_other_panel (panel);
+        chdir_other_panel (panel);
         break;
     case CK_PanelSelectSortOrder:
         panel_select_sort_order(panel);
